app/templates/fields.html

```diff
@@ -0,0 +1,75 @@
+<!doctype html>
+<html lang="en" ng-app="solarsenseApp">
+	<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+		<title>SolarSENSE</title>
+		<link rel="stylesheet" type="text/css" href="../static/css/bootstrap.min.css">
+		<link rel="stylesheet" type="text/css" href="../static/css/custom-style.css">
+		<link rel="stylesheet" type="text/css" href="../static/fontawesome/css/all.css">
+	</head>
+	<body ng-controller="FieldsCtrl">
+		<div class="loadingOverlay" ng-show="showLoader">
+			<div class="loader"></div>
+		</div>
+		<div class="header">
+			<div class="container">
+				<div class="logo pull-left"><h4 class="vertical" style="margin-left: 44px;color: #FFF;">SolarSENSE</h4></div>
+			</div>
+			<div class="clearfix"></div>
+			<div class="container">
```

Comment:
Update to use Global Nav Bar.

Reply:
Why is this Global nav bar a route to the flask server, why not just create it as a base template and have that included in the different html forms